examples/k8-manifests/README.md

```diff
@@ -0,0 +1,59 @@
+# Voting App deployed with raw kubernetes manifests
```

Comment:
@rberrelleza we could have two more folders here, one for helm and another one for cnab. this way, we provide examples about how to do CND with different deployment technologies.